### PR TITLE
ADK first pass-- in progress PR

### PR DESCRIPTION
**Pull request by andrewkern**
This PR is just a work in progress to show folks what I'm doing with the manuscript as I do it. Hoping to get some eyes/comments/suggestions here as I continue with commits over the next few days. No reason to merge yet.

### CODE REVIEW

**Comment by andrewkern** [comment]
not sure what happened with that merge... had to do with new figure version that got merged to master.  shouldn't hurt anything i think. 

meanwhile making progress on the manuscript. still whole chunks of results to add and write a discussion.

File: popSimManu.tex [hunk 1]
```diff
@@ -8,28 +8,53 @@
...
-\abstract{The explosion in population genomic data data demands ever increasingly complex modes of analysis. While
-the field has met this challenge to date, we have not yet created field-wide standards by which to measure the performance
-and accuracy of new tools. Here we describe a new resource, the PopSim database, that attempts to rectify this situation. This
-resource contains XXX...}
+\abstract{The explosion in population genomic data demands ever increasingly complex modes of analysis.
+More and more these analyses depend on sophisticated, often computationally expensive simulation. 
+To date, computational genetics researchers have been often forced to re-implement such simulations
+independently, leading to both wasted time in the duplication of effort and reductions in reproducibility.
+Population genetics, as a field, specifically lacks rigorous standard benchmarks by which new tools for inference might 
+be measured. Here we describe a new resource, the PopSim library, that attempts to rectify this situation.
```
Comment (jeromekelleher) [inline popSimManu.tex:53]: The resource is called stdpopsim though, isn't it?

File: popSimManu.tex [hunk 2]
```diff
@@ -43,39 +68,51 @@ \section*{Introduction}
...
-spatial patterns of genetic variation along the sequence. Dadi instead uses the
+methods use also differs. While \MSMC uses one to four whole genome sequences and leverages the
+spatial patterns of genetic variation along the sequence, \dadi  instead uses the
 site frequency spectrum (SFS), which summarizes the frequencies of SNPs in the
 sample, ignoring any information about the correlation structure (linkage
-disequilibrium) among SNPs.
+disequilibrium) among SNPs. Thus from the start these methods may not be 
+expected to perform equally well in all scenarios. \adk{I think we are focussed 
+a bit too much on benchmarking currently in the intro. Could be good to add more on efficiency
+and reproducibility}
```
Comment (jeromekelleher) [inline popSimManu.tex:106]: Agree with \adk comment here

**Review by jeromekelleher — APPROVED** [review]
Looks great, thanks @andrewkern. My main feedback is that we need to clarify what PopSim and stdpopsim are, but we can talk about that in today's call.

**Comment by ckyriazis** [comment]
looking good! i will work on adding in more two pop results ASAP.

**Comment by andrewkern** [comment]
Thanks Chris. I haven't gotten through that section yet, so hold off for about a day.

**Comment by andrewkern** [comment]
okay @jeromekelleher I think this is ready to merge. there is work to be done in the methods section yet, and there will be a lot of edits to do, but i think i've scaffolded  most of the important paragraphs.

**Comment by jeromekelleher** [comment]
OK, merging, thanks @andrewkern.

**Pull request merged**